src/test/java/teammates/test/driver/Retryable.java

```diff
@@ -0,0 +1,40 @@
+package teammates.test.driver;
+
+/**
+ * Represents a task that can be retried.
+ * @param <T> Result type.
+ * @param <E> Throwable type.
+ */
+public abstract class Retryable<T, E extends Throwable> {
```

Comment:
Can we update the class diagram of `Retryable`, `RetryableTask ` etc some where?